File: 프로그래머스/lv0/120836. 순서쌍의 개수/순서쌍의 개수.swift
import Foundation

func solution(_ n:Int) -> Int {
    var answer : Int = 0
    let root : Int = Int(sqrt(Double(n)))
    for i in 1...root {
        if (n % i == 0) {
            answer += 2
        }
    }
    return root * root == n ? answer - 1 : answer
}

File: 프로그래머스/lv0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {  
  var answer = 0;
  let sqrt = Math.ceil(Math.sqrt(n));
  for (let i = 1; i < sqrt; i++) {
    if (n % i === 0) answer += 2;
  }
  if (sqrt * sqrt === n) answer += 1;
  return answer;
}